cpp/locality/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+set(VOROPP_SOURCE_DIR
+    "${PROJECT_SOURCE_DIR}/extern/voro++/src"
+    )
+
+add_library(
+    _locality
+    OBJECT
+    AABB.h
+    AABBQuery.cc
+    AABBQuery.h
+    AABBTree.h
+    BondHistogramCompute.h
+    CMakeLists.txt
+    LinkCell.cc
+    LinkCell.h
+    NeighborBond.h
+    NeighborComputeFunctional.cc
+    NeighborComputeFunctional.h
+    NeighborList.cc
+    NeighborList.h
+    NeighborPerPointIterator.cc
+    NeighborPerPointIterator.h
+    NeighborQuery.cc
+    NeighborQuery.h
+    PeriodicBuffer.cc
+    PeriodicBuffer.h
+    RawPoints.h
+    Voronoi.cc
+    Voronoi.h
+
+    # For now, compile voro++ object in directly.
```

Comment:
There are CMake configuration files for voro++. Do they need to be edited, or just called correctly?

Reply:
They need to be edited. The current voro++ CMake files are not configured properly for building as a library in another code base, see the to-do list in the PR description.